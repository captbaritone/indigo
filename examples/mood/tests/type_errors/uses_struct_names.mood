struct Foo {
  x: i32,
}

pub fn test(): i32 {
  let foo: i32 = Foo { x: 100_i32 };
  200_i32
}