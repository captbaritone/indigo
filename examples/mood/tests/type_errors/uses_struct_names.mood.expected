-----------------
INPUT
----------------- 
struct Foo {
  x: i32,
}

pub fn test(): i32 {
  let foo: i32 = Foo { x: 100_i32 };
  200_i32
}
-----------------
OUTPUT
-----------------
Error: Expected "i32", got "Foo":
 --> uses_struct_names.mood:6:18

  | pub fn test(): i32 {
6 |   let foo: i32 = Foo { x: 100_i32 };
  |                  ^^^^^^^^^^^^^^^^^^ Expected "i32".
  |   200_i32
